src/components/EvidenceTile.tsx

```diff
@@ -59,7 +59,7 @@ interface ListProps {
 interface Props {
   residentId: string;
   id: string;
-  format: string;
+  format: string | undefined;
```

Comment:
In `[resident.Id]/index` we're defaulting this to `unknown` we can make this a `string`?

Reply:
It was a `string` previously but it was giving an error when assigning the format of the document as the function to extract the document extension is returning `string || undefined`

Reply:
Makes sense